### PR TITLE
parser-common: recognize events produced by SIGMA checkers

### PR DESCRIPTION
**Pull request by kdudka**

### CODE REVIEW

File: src/lib/parser-common.hh [hunk 1]
```diff
@@ -35,7 +35,8 @@
 
 #define RE_EVENT_GCC "(?:(?:(?:fatal|internal|runtime) )?[A-Za-z][A-Za-z0-9_-]+)(?:\\[[^ \\]]+\\])?"
 #define RE_EVENT_PROSPECTOR "(?:[A-Z]+[0-9]+\\[[a-z0-9-]+\\])"
-#define RE_EVENT RE_EVENT_GCC "|" RE_EVENT_PROSPECTOR
+#define RE_EVENT_SIGMA "(?:Sigma (?:main )?event)"
```
Comment (hanchuntao) [inline src/lib/parser-common.hh:38]: Should we need to define `remediation `?

Comment (kdudka) [reply src/lib/parser-common.hh:38]: `remediation` is already covered by `RE_EVENT_GCC` as all other events that do not contain spaces or any special delimiters.  This event specifically had already been also covered by a few tests.
```
% grep remediation: -r tests
```

**Comment by kdudka** [comment]
The failing CI job seems unrelated to the changes in this pull request.  I have created #160 for it.

File: src/lib/parser-common.hh (hunk 1 above)
Comment (hanchuntao) [reply src/lib/parser-common.hh:38]: ok, got it

**Review by hanchuntao — APPROVED** [review]

**Pull request merged**